### PR TITLE
Store resource ID in action's form_data attribute

### PR DESCRIPTION
**Pull request by osahyoun**

### CODE REVIEW

File: app/models/action.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Action < ActiveRecord::Base
+end
```
Comment (NealJMD) [inline app/models/action.rb]: talk about skinny models

**Comment by NealJMD** [comment]
this looks good but Im curious about the motivation for this

**Comment by osahyoun** [comment]
there have been times where the data team have wanted to refer back to the action record on champaign.

**Comment by osahyoun** [comment]
please merge if you're happy

**Pull request merged**